packages/console/src/pages/ApplicationDetails/index.tsx

```diff
@@ -92,7 +92,9 @@ function ApplicationDetails() {
       return;
     }
 
-    reset(data);
+    const { name, description, oidcClientMetadata, customClientMetadata, isAdmin } = data;
+
+    reset({ name, description, oidcClientMetadata, customClientMetadata, isAdmin });
```

Comment:
@xiaoyijun can you double check this update?  @wangsijie any particular reason ignoring other data fields?

Reply:
updated